libshvchainpack/src/abstractstreamreader.cpp

```diff
@@ -70,4 +70,13 @@ RpcValue AbstractStreamReader::read(std::string *error)
 	return ret;
 }
 
+ssize_t AbstractStreamReader::readCount() const
+{
+	ssize_t count = m_in.tellg();
```

Comment:
`tellg` does not return ssize_t. I recommend using `auto`
```suggestion
	auto count = m_in.tellg();
```